src/main/kotlin/com/exactpro/th2/codec/oracle/logminer/LogMinerTransformer.kt

```diff
@@ -138,15 +136,430 @@ class LogMinerTransformer(private val config: LogMinerConfiguration) : IPipeline
         }
     }
 
+    // TODO: handle parsing exception
+    class InsertListener(private val builder: MapBuilder<String, Any?>, private val prefix: String) : PlSqlParserBaseListener() {
+        private val columNames: Queue<String> = LinkedList()
+        private var stage = Stage.BEGIN
+        private val expressionHolder: MutableList<Expression> = mutableListOf()
+
+        override fun visitErrorNode(node: ErrorNode) {
+            super.visitErrorNode(node)
+        }
```

Comment:
Do we need to override this if we just call super method?

Reply:
I began to experiment with capture parse error from antlr, but haven't completed yet